bedrock/firefox/templates/firefox/mobile.html

```diff
@@ -42,6 +42,7 @@
 
               <div class="header-product-ctas">
                 <a href="#mobile-download-buttons-firefox" class="button quantum get-firefox" data-product="firefox" data-button-name="Send to SMS Modal - Firefox Mobile">{{ _('Get Firefox') }}</a>
+                <br>
```

Comment:
By no means a blocker, but could you remove these `<br>`'s if you used a combination of `display: block;` and `max-width` on the buttons? I haven't tested this, so feel free to ignore if you already tried this path 👍

Reply:
Going the `max-width` route does indeed work, but I opted to allow the button to be its natural width (based on localized text) rather than dictate what that width should always be in CSS. Does result in the added `<br>` tag though.